app/client/components/SpeedIndicator.js

```diff
@@ -27,7 +33,7 @@ export default class SpeedIndicator extends React.Component {
         }
 
         return (
-            <div style={ speedStyle }>
+            <div style={ speedStyle } onClick={(e) => {this.speedClicked(e)}}>
```

Comment:
It is throwing me an error when I click on the component "Uncaught TypeError: Cannot read property 'props' of null"